scripts/utils/verify/writeJson.ts

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs');
+import hre from "hardhat";
+
+import { main as writeInfo } from "./writeInfo";
+import { contractInfo } from "./writeInfo";
+import { contractData, PromissoryNoteTypeBn, PromissoryNoteTypeLn } from "../../deploy/deploy";
+import { SECTION_SEPARATOR } from "../bootstrap-tools";
+
+export interface deploymentData {
+    [contractName: string]: contractData | PromissoryNoteTypeBn | PromissoryNoteTypeLn
+}
+
+export async function main(
+    assetVaultAddress: string,
+    feeControllerAddress: string,
+    borrowerNoteAddress: string,
+    lenderNoteAddress: string,
+    repaymentContAddress: string,
+    whitelistAddress: string,
+    vaultFactoryAddress: string,
+    loanCoreAddress: string,
+    originationContAddress: string
+  ): Promise<void> {
+            const timestamp = new Date().getTime() * 1000
```

Comment:
Formatting very strange in this file - looks like some lines need to be outdented and some indendented. Try just using prettier - `yarn prettier --write scripts/verify/writeJson.ts`